src/main/java/com/google/sps/servlets/MyTaskServlet.java

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.gson.Gson;
+import com.google.sps.task.Task;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that fetch all saved tasks whose owner is the current user. */
+@WebServlet("/mytasks")
+public class MyTaskServlet extends HttpServlet {
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // "Leonard" will be replaced with the user's accound id in the future
+    String keyword = request.getParameter("keyword");
+    Filter ownerFilter = new FilterPredicate(keyword, FilterOperator.EQUAL, "Leonard");
```

Comment:
We'll also be filtering on the "owner" field, right? Should we be doing that already?

Reply:
Ahh actually it's my naming problem. The filter is actually different depending on the input keyword. The doGet function mainly serves the GET request sent from the Need Help page and the Offer Help page. The Need Help play displays all the tasks whose owner is current user so the GET request sent by that page will have a keyword of "Owner". Then the filter will filter on the "owner" field. For the Offer help page, it will send a GET request with keyword "Helper". Thus the filter will filter on the "helper" field. So I would rename the `Filter` element in the new commit.